packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -110,6 +114,16 @@ export class MetadataEditor extends ReactWidget {
     this.update();
   }
 
+  private isValueEmpty(schemaValue: any): boolean {
+    return (
+      schemaValue === undefined ||
+      schemaValue === null ||
+      schemaValue === '' ||
+      schemaValue === [] ||
```

Comment:
This is probably not gonna work since an array is an object, if we do [] === [] we get `false`
To be safe, we could do `Array.isArray(schemaValue) && schemaValue.length`

Reply:
nice catch, I'll make that change